tools/appveyor.psm1

```diff
@@ -326,9 +326,15 @@ function Invoke-AppVeyorTest
     Write-Host -Foreground Green 'Upload FullCLR test results'
     Update-AppVeyorTestResults -resultsFile $testResultsFileFullCLR
 
-    ## CodeCoverage
-    $env:CodeCoverageOutput = Split-Path -Parent (Get-PSOutput -Options (New-PSOptions -Configuration CodeCoverage))
- 
+    if(Test-DailyBuild)
+    {
+        ## Publish code coverage build, tests and OpenCover module to artifacts, so webhook has the information.
```

Comment:
> artifacts

I think the comment is missleading. AppVeyor has separate tabs/concepts for Tests and Artifacts. It seems that here we use `Update-AppVeyorTestResults` => tests tab and don't publish the artifacts (in the appveyor terminology).